include/engine/guidance/route_step.hpp

```diff
@@ -162,6 +163,8 @@ inline RouteStep &RouteStep::ElongateBy(const RouteStep &following_step)
                          following_step.intersections.begin(),
                          following_step.intersections.end());
 
+    /// @todo Process is_segregated flag
```

Comment:
what is this todo referring to? If we want to go ahead with a merge, this should be made into an issue and referred to or removed.